packages/server/graphql/queries/helpers/getPublicScoredTemplates.ts

```diff
@@ -2,19 +2,21 @@ import db from '../../../db'
 import getTemplateScore from '../../../utils/getTemplateScore'
 
 const getPublicScoredTemplates = async (
-  templates: {createdAt: Date; id: string; isStarter?: boolean}[]
+  templates: {createdAt: Date; id: string; isStarter?: boolean; isFree?: boolean}[]
 ) => {
   const sharedTemplateIds = templates.map(({id}) => id)
   const sharedTemplateEndTimes = await db.readMany('endTimesByTemplateId', sharedTemplateIds)
   const scoreByTemplateId = {} as {[templateId: string]: number}
   templates.forEach((template, idx) => {
-    const {id: templateId, createdAt, isStarter} = template
+    const {id: templateId, createdAt, isStarter, isFree} = template
     const endTimes = sharedTemplateEndTimes[idx]!
+    const isFreeBonus = isFree ? 1000 : 0
```

Comment:
Hey @mattkrick, we have the `isStarter` field in `TemplateScale` from https://github.com/ParabolInc/parabol/pull/4220. It looks like all public templates have `isStarter === true`, but as `getPublicScoredTemplates` is checking if `isStarter` is true, I imagine it wasn't intended to be this way?

Perhaps it was meant to indicate which templates should feature higher up the list?

Now that we're restricting templates, templates that are accessible to everyone have the `isFree` field. We want those to feature at the top of the list.

I want to check:
1) That my understanding of the `isStarter` templates is correct 
2) Whether you think adding a large bonus to the `isFree` templates is the right move?
3) Should `isStarter` be removed if it's true for all public templates?

Reply:
@mattkrick just bumping this question

Reply:
yep, you got it. the old logic was to allow folks to mark their custom templates as public & then as the templates gain traction, other folks start to use them & we create a kind of marketplace for templates. We stopped building that (https://github.com/ParabolInc/parabol/issues/4099), but the groundwork is still there.

Since moderating user-generated templates is so difficult & that's no longer the direction we're heading, I'd say it's probably safe to remove that type of logic entirely. `getPublicScoredTemplates` could be removed &  `publicTemplates` could simply return everything created by parabol (ie `teamId: 'aGhostTeam'`)

adding `isFree` sounds like a good move!

Reply:
Sounds good! I've implemented that change. Now that we're no longer moving in the user-generated templates direction, shall we remove the "share with the world" option as custom templates can only be shared with the team/org?

![share-with-world](https://user-images.githubusercontent.com/39854876/201917574-cc635370-b683-4908-ac33-74cda6cb092d.png)